src/Network/Haskoin/Store/Data.hs

```diff
@@ -507,6 +564,8 @@ instance BinSerial StoreInput where
                 StoreCoinbase {}               -> B.empty
```

Comment:
We don't need an instance of `BinSerial` for `StoreInput`. It can be completely removed.